controllers/auth.js

```diff
@@ -59,15 +59,23 @@ const sendResetPasswordEmail = async (req, res) => {
 
   await authService.sendResetPasswordEmail(email)
 
-  res.sendStatus(204)
+  res.status(204).json()
```

Comment:
just `end()`, you don't send any data, so you don't need this. please, fix in other cases

Reply:
fixed